assets/js/modules/globalNav.js

```diff
@@ -144,8 +149,11 @@ define([
      * @returns {Boolean} true if spec shoud be skipped.
      */
     var skipSpec = function(navListCat, obj) {
+        obj = obj || {};
         // obj["cat"] is an array; if cat has needed value
-        var inArray = !!obj["tag"] && obj["tag"].indexOf(navListCat) > -1;
+        var inArray = !!obj["tag"] && typeof(navListCat) === "string"
```

Comment:
If not string, then it could be an object/number etc, which will fail or produce unexpected output.